docs/usage/self-hosted-configuration.md

```diff
@@ -690,6 +690,12 @@ Renovate will then create branches on the fork and opens Pull Requests on the pa
 !!! note
     Forked repositories will always be skipped when `forkToken` is set, even if `includeForks` is true.
 
+## gitCredentialPassing
```

Comment:
It looks like the .gitcredentials file in the home directory will be modified by Renovate (and not reverted at the end of a job). Both these should be in the docs